menas/ui/components/dataset/conformanceRule/ConformanceRuleForm.js

```diff
@@ -374,13 +374,8 @@ class MappingConformanceRuleForm extends ConformanceRuleForm {
   }
 
   hasValidJoinConditions(fieldValue = []) {
-    let isValid = fieldValue.length >= 1;
-
-    if (!isValid) {
-      sap.m.MessageToast.show("At least 1 join condition is required.");
-    }
-
-    return isValid
+    // empty join conditions are allowed
```

Comment:
How hard would it be to check either for >= 1 join conditions or >= 1 field equals filters?

Reply:
Or why not remove the method and its uses?

Reply:
@Zejnilovic: I have done it on UI in https://github.com/AbsaOSS/enceladus/pull/1970/commits/c35ab4c15c93091d4b4dc6c10533553094a2da7a . It looks like this:

![join-condition-or-filter](https://user-images.githubusercontent.com/4457378/140078402-2cb7df47-4728-45a3-8ea4-aea404ddad75.png)

(@AdrianOlosutean, could be, but now it is used again)